### DISCUSSION TITLE
Where are the other updates?

### DISCUSSION THREAD

**Discussion by RikisuT** [ShankarBUS/ModernFlyouts#846] [General]
Are you guys working on multiple versions at once? cause I cant find versions 9.4 and onwards...

**Comment by Samuel12321** [comment]
Sort of, the next release was originally scheduled to be 0.9.4. Unfortunately a bug in the host prevented it being released. Work slowly continued and we eventually decided to rename the next release 0.10 due to the larger changes that had been added. However the host bug still hasn't yet been fixed. So we have 0.9.4 which evolved into 0.10 ready to release, except for 1 critical bug.

Parallel to this a subset of our team about a month ago began work on a major new release currently called 0.11. This new release is set to rebuild major components of ModernFlyouts, add a new settings experience, new tray flyouts, align styles to windows 11 and more.

So to sum up a complex answer to your question, the reason you can't find 0.9.4 or later is that 0.9.3 is currently the latest **released** version. And we are working on 2 new releases.

**Comment by netrom-live** [comment]
Any update on this? As far as I can see 0.9.3 is still currently the latest released version, excluding the beta-version with the expired certificate.

**Comment by hnavia** [comment]
IS this project still active? It seems there hasn't been any updates in a very long time.

**Comment by MightyM8** [comment]
Would love to know if this is still in progress (updates mentioned above), or has been abandoned? I'm loving the current (Windows Store, v0.9.3), just wondering about future developments and patches as I have had to employ other app's to help fix some of the other issues (i.e. default windows 11 flyout still appears, etc).

**Comment by fillwithjoy1** [comment]
https://github.com/ModernFlyouts-Community/ModernFlyouts/issues/1153#issuecomment-1407568509
